commons/ui/src/main/kotlin/com/teacherworkout/commons/ui/model/Lesson.kt

```diff
@@ -0,0 +1,15 @@
+package com.teacherworkout.commons.ui.model
+
+import androidx.annotation.DrawableRes
+
+//TODO: refactor to match the model from the GraphQL
```

Comment:
In my opinion, these comments are not necessary as the people who will help with this will do these reactors for sure

Reply:
I'm also with you on this ;p